pkg/agentctl/samples.go

```diff
@@ -0,0 +1,131 @@
+package agentctl
+
+import (
+	"time"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/pkg/timestamp"
+	"github.com/prometheus/prometheus/promql/parser"
+	"github.com/prometheus/prometheus/tsdb/record"
+	"github.com/prometheus/prometheus/tsdb/wal"
+)
+
+type Series struct {
+	Labels labels.Labels
+	MinTS  time.Time
+	MaxTS  time.Time
+	Count  int64
+}
+
+// FindSamples TODO
```

Comment:
Should this comment be removed? :)

Reply:
It should :slightly_smiling_face: 

// TODO remove me